kernel/base/src/main/java/com/twosigma/beakerx/kernel/commands/MagicCommand.java

```diff
@@ -230,8 +234,8 @@ private Boolean containsSingleWildcardSymbol(String path) {
     try {
 
       Map<Path, String> paths = Files.list(Paths.get(pathWithoutWildcards))
-              .filter(path -> path.toString().toLowerCase().endsWith(".jar"))
-              .collect(Collectors.toMap(p -> p, o -> o.getFileName().toString()));
+                                     .filter(path -> path.toString().toLowerCase().endsWith(".jar"))
+                                     .collect(Collectors.toMap(p -> p, o -> o.getFileName().toString()));
 
       if (paths == null || paths.isEmpty()) {
         throw new IllegalStateException("Cannot find any jars files in selected path");
```

Comment:
as we discussed, this exception is inappropriate for user errors.
use a conditional.